configure.py

```diff
@@ -1041,6 +1041,7 @@ def find_headers(repodir, excluded_dirs):
                 'service/raft/raft_group0.cc',
                 'direct_failure_detector/failure_detector.cc',
                 'service/raft/raft_group0_client.cc',
+                'task_manager/task_manager.cc',
```

Comment:
Wouldn't `tasks` be a better directory name? How about also adding a namespace which is used in all files of this dir?